.github/actions/setup_ubuntu/action.yml

```diff
@@ -10,6 +10,13 @@ runs:
     - name: Install Dependencies
       working-directory: ${{ inputs.cwd }}
       run: |-
+
+        # For the datadog cli, it must be installed via python
```

Comment:
Do we still need datadog? 🤔

Reply:
I don't know, i want to restore CI asap.

Reply:
let's create a ticket to evaluate the usage of datadog. Thanks for looking into it 👍